src/Accounts/Authentication/Authentication/TokenCache/PowerShellTokenCacheProvider.cs

```diff
@@ -174,7 +174,7 @@ public virtual IPublicClientApplication CreatePublicClient(string authority = nu
             if (AzureSession.Instance.TryGetComponent<IConfigManager>(nameof(IConfigManager), out var config)
                 && config.GetConfigValue<bool>(ConfigKeys.EnableLoginByWam))
             {
-                builder = builder.WithBrokerPreview();
+                builder = builder.WithBroker();
```

Comment:
Please check if we should use the override with parameter

Reply:
The function is just to enable WAM. Another interface to customize WAM `public PublicClientApplicationBuilder WithBroker(BrokerOptions brokerOptions)` is only supported above NET6
https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/25a286e3d963ccd913d67dcc0caa16047e9a65bb/src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs#L223